bottom_sheet/README.md

```diff
@@ -0,0 +1,16 @@
+# project
+
+A new Flutter project.
```

Comment:
Provide description and demo gif of the example app. Check other example apps for reference.